copilot-instructions.md

```diff
@@ -0,0 +1,408 @@
+# GitHub Copilot Instructions for Sitecore Content SDK
+
+## Project Overview
+
+This is the **Sitecore Content SDK** - a comprehensive TypeScript/JavaScript SDK for building modern web applications with Sitecore XM Cloud. The project provides core functionality, React components, Next.js integration, and CLI tools for scaffolding applications.
+
+### Tech Stack
+- **Language**: TypeScript (Node LTS)
+- **Runtime**: Node LTS or higher
+- **Build**: `tsc` -> `dist/`, templates bundled via `scripts/build-templates.ts`
+- **Testing**: Mocha + Sinon + Chai, coverage via `nyc`
+- **Lint/format**: ESLint + Prettier
+- **Package Manager**: Yarn 3.1.0
+
+### Core Packages
+- `@sitecore-content-sdk/core` - Core SDK functionality
+- `@sitecore-content-sdk/react` - React components and hooks
+- `@sitecore-content-sdk/nextjs` - Next.js integration and middleware
+- `@sitecore-content-sdk/cli` - CLI tools and scaffolding
+- `@sitecore-content-sdk/create-content-sdk-app` - Scaffolding CLI & templates
+
+## Repository Structure
+
+```
+content-sdk/
+├── packages/
+│   ├── create-content-sdk-app/    # Scaffolding CLI & templates
+│   ├── core/                      # Core SDK functionality
+│   ├── nextjs/                    # Next.js-specific SDK
+│   ├── react/                     # React components
+│   └── cli/                       # Additional CLI tools
+├── .cursor/rules/                 # AI coding guidance
+└── samples/                       # Example applications
+```
+
+### Key Directories
+- **Sources**: `src/**` (within create-content-sdk-app package)
```

Comment:
Hm, why do we need to mention "create-content-sdk-app" package here explicitly? I believe it's about all our packages